tests/wpunit/MenuItemConnectionQueriesTest.php

```diff
@@ -834,4 +834,48 @@ public function testQueryMenuItemsWithParentDatabaseIdSetToZero() {
 			$this->assertNull( $node['parentId'] );
 		}
 	}
+
+	public function testMenuItemsOrderOnlyAppliedWhenOrderByIsSet() {
+    $menu_location = 'test-order-location';
+    register_nav_menu($menu_location, 'Test Order Location');
+    WPGraphQL::clear_schema();
+
+    // Create menu items
+    $created = $this->create_menu_items('order-test-menu', $menu_location, 3);
+    $query   = $this->getQuery();
+
+    // CASE 1: Query WITHOUT `orderby`
+    $variables = [
+        'where' => [
+            'location' => WPEnumType::get_safe_name($menu_location),
+        ],
+    ];
+    $actual_without_orderby = $this->graphql(compact('query', 'variables'));
```

Comment:
The query is being run _with_ the variables, but is set as `$actual_without_orderby`. Did you mean to execute the query without the variables as a baseline first as the variable name `$actual_without_orderby` implies?

Reply:
scratch that. Now that I give it a second look, I see. We're passing the location but letting the `orderby` remain default. Forget my previous comment.